__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const request = require('supertest');
+
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
+
+describe('testing error for any incorrect path', () => {
+    test('GET /apo', () => {
+        return request(app)
+        .get('/apo')
+        .expect(404)
+        .then(({body: {message}}) => {
+            expect(message).toBe('Path not found :( Try again.')
```

Comment:
when we're throwing our own error objects it's good to use 'msg' instead of 'message' as it clarifies to us this is our own error object instead of something more 'official' thrown by psql or node etc.